pages/spells/spells-table.vue

```diff
@@ -58,37 +59,40 @@
           </tr>
         </thead>
         <tbody>
-          <tr v-for="spell in spellsListed" :key="spell.name">
+          <tr v-for="spell in spellsListed" :key="spell.slug">
             <th>
               <nuxt-link
                 tag="a"
                 :params="{ id: spell.slug }"
                 :to="`/spells/${spell.slug}`"
+                class="mr-2"
               >
                 {{ spell.name }}
               </nuxt-link>
               <source-tag
                 v-if="
                   spell.document__slug && spell.document__slug !== 'wotc-srd'
                 "
-                class=""
+                class="ml-0 hide-mobile"
                 :title="spell.document__title"
                 :text="spell.document__slug"
               />
             </th>
-            <td>{{ spell.school }}</td>
+            <td>{{ capitalize(spell.school) }}</td>
             <td>{{ spell.level_int }}</td>
             <td class="hide-mobile">
               {{ spell.components }}
             </td>
             <td class="hide-mobile">
               <span
-                v-for="(spellclass, index) in spell.dnd_class"
+                v-for="(spellclass, index) in spell.spell_lists"
                 :key="spellclass"
-                ><span class="dnd_class" @click="filterByClass(spellclass)">{{
-                  spellclass
+                ><span class="spell_lists" @click="filterByClass(spellclass)">{{
+                  capitalize(spellclass)
                 }}</span
-                ><span v-if="index + 1 < spell.dnd_class.length">, </span></span
+                ><span v-if="index + 1 < spell.spell_lists.length"
+                  >,
+                </span></span
               >
```

Comment:
Hopefully it is a little easier to parse now? ESLint or prettier did some funky stuff to some of our existing code at some point.

I do want to leave it for now since I'm trying not to add more risk to this PR, but that would also be a great follow-on.